src/state/apis/swapper/helpers/validateQuoteRequest.ts

```diff
@@ -29,8 +29,8 @@ export const validateQuoteRequest = ({
     return [{ error: TradeQuoteRequestError.NoConnectedWallet }]
   }
 
-  const walletSupportsSellAssetChain = walletSupportedChainIds.includes(sellAsset.chainId)
-  const walletSupportsBuyAssetChain = walletSupportedChainIds.includes(buyAsset.chainId)
+  const walletSupportsSellAssetChain = walletConnectedChainIds.includes(sellAsset.chainId)
+  const walletSupportsBuyAssetChain = walletConnectedChainIds.includes(buyAsset.chainId)
```

Comment:
And ditto for this guy, LGTM but untestable, given we won't be able to get a quote for an unsupported buy asset without a manual receive address